src/App.tsx

```diff
@@ -1,23 +1,10 @@
 import React from 'react';
-import { CarouselSection } from './components/carousel-section/carousel-section';
-import { Hero } from './components/hero/hero';
-import { Banner } from './components/banner/banner';
-import { Menu } from './components/menu/menu';
-import { Separator } from './components/separator/separatos';
-import { Footer } from './components/footer/footer';
+import { SectionImmobile } from './components/carousel-section-immobile/carousel-section-immobile';
 
 export function App() {
   return (
     <div>
-      <Menu />
-      <Separator size="large" />
-      <Hero />
-      <CarouselSection title={'Encontre o imóvel ideal para o seu estilo de vida!'} type="category" />
-      <Separator size="large" />
-      <Banner />
-      <CarouselSection title={'Lançamentos que você pode se interessar!'} type="property" />
-      <Separator size="large" />
-      <Footer />
+      <SectionImmobile />
```

Comment:
por que excluiu toda a home? Pode deixar e implementar essa sessao onde ela vai ficar

Reply:
A home foi separada em outro componente em outro pr. Pensei poderia limpar o App.tsx

Reply:
se vc for colocar em outro componente a home é de boas, mas coloca o componente onde ele ficaria independentemente